packages/eslint-plugin-nimble/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@ni-private/eslint-plugin-nimble",
+  "version": "0.0.1",
+  "private": true,
+  "description": "Common ESLint configuration for the Nimble repository",
```

Comment:
We should add an explicit `"type": "commonjs"` as that is what this package currently is.
At least I believe eslint doesn't support plugins that are type module.